lib/apr/notifications.ex

```diff
@@ -3,69 +3,70 @@ defmodule Apr.Notifications do
   require Logger
 
   def receive_event(event, topic, routing_key) do
-    event
-    |> slack_message(topic, routing_key)
-    |> post_message(topic, routing_key)
+    get_subscriptions(topic, routing_key)
+    |> Enum.map(&{&1, slack_message(&1, event, topic, routing_key)})
+    |> Enum.map(&post_message(&1))
   end
 
-  def slack_message(event, topic_name, routing_key) do
+  defp get_subscriptions(topic, routing_key) do
```

Comment:
since this just calls the other method, i think we should just remove this method and call `Subscriptions.get_topic_subscribers` directly on line 6.